docs/guide/slots.md

```diff
@@ -51,9 +55,13 @@ To render a `renders_many` slot, iterate over the name of the slot:
 <% end %>
 ```
 
+The link to "My blog" will render inside the `<h1>` tags, and links to each `BlogPost` will be rendered beneath.
+
 ## Component slots
 
-It's also possible to have a slot be a ViewComponent itself by passing in a second argument to `renders_one` and `renders_many`:
+Slots can also render other components. Pass the name of the component as the second argument. Component slots are
+useful when you want to render a specific component, but still allow full control of the arguments that are passed to
+it.
```

Comment:
```suggestion
Slots can also render other components by passing the name of the component as the second argument:
```

> enabling useful when you want to render a specific component, but still allow full control of the arguments that are passed to it.

I'm not sure what you mean by `full control` here. Can you explain another way?

Reply:
By "full control", I'm getting at the idea that lambda slots allow you to set defaults for, restrict, and modify the arguments passed to components, whereas component slots call the component with exactly the arguments you give it.

Reply:
I think that explanation would be nice to incorporate in place of `full control` 😄